README.md

```diff
@@ -80,6 +80,11 @@ unclassified:
       - elastic:
           kibanaBaseUrl: "http://localhost:5601"
           name: "Elastic Observability"
+          displayKibanaDashboardLink: true
+          elasticLogsBackend:
+            elasticLogsBackendWithJenkinsVisualization:
```

Comment:
Can we also document the `elasticLogsBackendWithoutJenkinsVisualization`? It doesn't take any parameter, it should be an easy one

Reply:
Can we have more than one entry? Otherwise, how do we document it?
We can try in a follow up for simplicity?

Reply:
it's exclusive, don't we have a "reference guide" on JCasC config?

Reply:
I don't understand your question :/

https://github.com/jenkinsci/opentelemetry-plugin#configuration-as-code is the current reference, and since it's `exclusive` I don't know a way to simple add an example. 

There is a reference for some examples in `See the [jcasc](https://github.com/jenkinsci/opentelemetry-plugin/blob/master/src/test/resources/io/jenkins/plugins/opentelemetry/jcasc) folder with various samples.`

If that's what you mean we can create a test set and its UT ?

Reply:
I thought we had a "reference guide of otel plugin JCasC config" in a place like https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos .

I was thinking of each configuration scenario to be documented:
* Jaeger for traces
* Zipkin for traces
* Elastic just for traces and metrics
* Elastic for traces, metrics, and logs with the 2 scenarios
   * Exclusively logs visualization in Elastic Kibana
   * Logs visualization both in Elastic Kibana and through Jenkins

Would that make sense?